lib/developer.js

```diff
@@ -78,6 +78,7 @@ function developer (opts) {
     }, opts);
 
     initialRequest(opts)
+      .then(getListWithPermissions(opts))
```

Comment:
Sure, I totally agree with you! I just wanted to be sure that we'd need to refactor the actual code to achieve it and I was not missing any obvious way to get it working. 

Since almost all methods are using this way to search the apps and controll the `savedApp` outside the `getParseList` and we'd need to modify them too, do you think is it worth to open a new base branch only for this refactoring?